custom_components/tahoma/const.py

```diff
@@ -38,8 +38,9 @@
     "SunSensor": "sensor",
 }
 
-# TaHoma internal device states
 CORE_ON_OFF_STATE = "core:OnOffState"
 
-# Commands
 COMMAND_OFF = "off"
+COMMAND_ON = "on"
+
+ICON_WEATHER_WINDY = "mdi:weather-windy"
```

Comment:
Tbh, if an ICON is used in two integrations I would just duplicate it for readability and manageability.

Reply:
done